macros/models/_sector/referral/platforms/fabric_referral_rewards.sql

```diff
@@ -0,0 +1,45 @@
+{% macro fabric_referral_rewards(
+    blockchain
+    ,SubscriptionTokenV1_evt_ReferralPayout
+    ,SubscriptionTokenV1Factory_call_deploySubscription
+    )
+%}
+
+with currency_contract_info as (
+    select
+        erc20TokenAddr,
+        contract_address
```

Comment:
You need to take the `output_0` here.
The contract_address is the factory itself

Reply:
Thanks, these two comments makes the CI green.